Neither of these drivers does anything on-GPU, so, no.

diff -up mesa-20130508/src/gallium/targets/Makefile.am.jx mesa-20130508/src/gallium/targets/Makefile.am
--- mesa-20130508/src/gallium/targets/Makefile.am.jx	2013-05-08 13:23:41.000000000 -0400
+++ mesa-20130508/src/gallium/targets/Makefile.am	2013-05-08 14:59:50.897835932 -0400
@@ -78,10 +78,6 @@ endif
 if HAVE_ST_XVMC
 SUBDIRS += xvmc-r300
 endif
-
-if HAVE_ST_VDPAU
-SUBDIRS += vdpau-r300
-endif
 endif
 
 if HAVE_GALLIUM_R600
@@ -142,10 +138,6 @@ endif
 if HAVE_ST_XVMC
 SUBDIRS += xvmc-softpipe
 endif
-
-if HAVE_ST_VDPAU
-SUBDIRS += vdpau-softpipe
-endif
 endif
 
 if NEED_GALLIUM_LOADER
